chain/indexer.go

```diff
@@ -626,6 +646,9 @@ func (t *TipSetIndexer) stateChangedActors(ctx context.Context, old, new cid.Cid
 }
 
 func (t *TipSetIndexer) runMessageProcessor(ctx context.Context, p MessageProcessor, name string, ts, pts *types.TipSet, emsgs []*lens.ExecutedMessage, blkMsgs []*lens.BlockMessages, results chan *TaskResult) {
+	ctx, span := otel.Tracer("").Start(ctx, fmt.Sprintf("TipSetIndexer.%s", name))
```

Comment:
I think it could be worth making this name more specific such as `"TipSetIndexer.MessageProcessor.%s"` (and the same for each of the run processor functions below)